src/client/components/Title/TitleDesc.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TitleDesc.styles.css'
+
+const TitleDesc = ({title,description}) =>{ return (<div> <h3 className="title">{title}</h3>
+                                           <p className="description">{description}</p></div> ); };
+
+export default TitleDesc
+
+TitleDesc.propTypes = {
+title: PropTypes.string.isRequired,
+description: PropTypes.string.isRequired,
+};
```

Comment:
Proptypes should be assigned before export. 
```suggestion
TitleDesc.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
};

export default TitleDesc
```